Makefile

```diff
@@ -0,0 +1,7 @@
+.PHONY: changelog
+changelog:
+	./scripts/release-notes.py --exclude-dependabot --repo jaeger-ui --point-markdown-char '-'
```

Comment:
could we just curl/wget this from the main repo ?

https://raw.githubusercontent.com/jaegertracing/jaeger/main/scripts/release-notes.py

Reply:
Yup, that's what I plan to do, just wanted to get your thoughts on whether we should do the same for `./scripts/draft-release.py` as well before I make that change.

Reply:
Sure